packages/tocco-ui/src/Layout/LayoutContainer.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import StyledLayoutContainer from './StyledLayoutContainer'
+import {layoutPropTypes, stylingLayout} from '../utilStyles'
+
+/**
+ * Use <Layout.Container/> to apply grid and gutter on <Layout.Box>.
+ * Define how many cells are displayed per row and breakpoint.
+ */
+class LayoutContainer extends React.Component {
+  state = {
+    containerWidth: undefined
```

Comment:
isnt undefined the default? I think you can remove this?

Reply:
I still need to declare `state = {}` in order that `containerWidth: this.state.containerWidth` does not fail on line 38.